BattleNetwork/overworld/bnOverworldActor.cpp

```diff
@@ -603,9 +603,9 @@ std::string Overworld::Actor::FindValidAnimState(const Direction& dir, const Mov
   auto str = MovementAnimStrPrefix(state) + "_" + DirectionAnimStrSuffix(Isometric(dir));
 
   if (anim.HasAnimation(str) == false) {
-    const auto& [ud, lr] = Split(Isometric(dir));
+    auto [ud, lr] = Split(Isometric(dir));
 
-    std::map<Direction, std::initializer_list<std::tuple<bool, Direction, MovementState>>> attempts = {
```

Comment:
from cppref:

> Initializer lists may be implemented as a pair of pointers or pointer and length. Copying a std::initializer_list does not copy the underlying objects. 

Yikes. This should really be highlighted in all the intro material with C++. I've never known about this and had to read up on it myself.